services/src/main/java/org/keycloak/authentication/authenticators/resetcred/ResetOTP.java

```diff
@@ -33,12 +48,97 @@ public class ResetOTP extends AbstractSetRequiredActionAuthenticator implements
 
     public static final String PROVIDER_ID = "reset-otp";
 
+    private static final String ACTION_ON_OTP_RESET_FLAG = "action_on_otp_reset_flag";
+    private static final String REMOVE_NONE = "Remove none";
+    private static final String REMOVE_ONE = "Remove one";
+    private static final String REMOVE_ALL = "Remove all";
+
+    private static final List<ProviderConfigProperty> config;
+
+    static {
+        ProviderConfigurationBuilder builder = ProviderConfigurationBuilder.create();
```

Comment:
We should avoid doing static init for providers that are not necessarily used at runtime and save a bit more memory footprint during startup (and potentially at runtime as well).

It is OK to just create this at the `getConfigProperties` method.

I know a other providers are doing the same and we need to fix them too in the future.

Reply:
Fixed. The config is now initialized inside the getConfigProperties() method.